ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/clientlibs/less/button.less

```diff
@@ -1,105 +1,175 @@
-/* button style */
-.btn-primary .cmp-button{
-         background: #2979be;
-        border : 1px solid #165fab;
-        color:#fff; 
-        padding:10px 20px; 
-        font-size:14px;
-        cursor: pointer;
-        display: inline-block;
-		text-align: center;
-		text-decoration: none;
-		width: 250px;
-		box-sizing: border-box;
-        &:hover{
-            background: #0c4f95;
-        }
-}
-
-.btn-primary__disabled .cmp-button{
-         background: #9fbbd9;
-        border : 1px solid #9fbbd9;
-        color:#fff; 
-        padding:10px 20px; 
-        font-size:14px;
-        cursor: pointer;
-        text-align: center;
-		text-decoration: none;
-		width: 250px;
-		box-sizing: border-box;
-		display: inline-block;
-        &:hover{
-            cursor: auto;
-        }
-}
-
-.btn-secondary .cmp-button{
-        border: 1px solid #666666; 
-        color: #666666; 
-        padding:10px 20px; 
-        font-size:14px;
-        cursor: pointer;
-        text-align: center;
-		text-decoration: none;
-		width: 250px;
-		box-sizing: border-box;
-        background : #ffffff;
-		display: inline-block;
-        &:hover{
-            background: #666;
-    	    color : #ffffff;
-        }
-}
-
-.btn-secondary__disabled .cmp-button{
-       border: 1px solid #666666; 
-        color: #666666; 
-        padding:10px 20px; 
-        font-size:14px;
-        cursor: pointer;
-        text-align: center;
-		text-decoration: none;
-		width: 250px;
-		box-sizing: border-box;
-        background : #ddd8d829;
-		display: inline-block;
-        &:hover{
-            cursor: auto;
-        }
-}
-
-.btn-ternary .cmp-button{
-        border: 1px solid #ff471e; 
-        color: #ffffff; 
-        padding:10px 20px; 
-        font-size:14px;
-        cursor: pointer;
-        text-align: center;
-		text-decoration: none;
-		width: 250px;
-		box-sizing: border-box;
-        background : #ff471e;
-		display: inline-block;
-}
-
-.btn-ternary__disabled .cmp-button{
-        border: 1px solid #efaa9b; 
-        color: #ffffff; 
-        padding:10px 20px; 
-        font-size:14px;
-        cursor: pointer;
-        text-align: center;
-		text-decoration: none;
-		width: 250px;
-		box-sizing: border-box;
-        background : #efaa9b;
-		display: inline-block;
-        &:hover{
-	  cursor: auto;
-    }
+.btn {
```

Comment:
@ReenaOmana : who from UI has made these changes ?

Reply:
Ayush had made the changes.